cmd/agent/main.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019 Red Hat and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Main package for the Nimbess agent executable.
+package main
+
+import (
+	"flag"
+	"path/filepath"
+
+	"io"
+	"os"
+	"sync"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/trozet/nimbess-agent/pkg/agent"
```

Comment:
Sure this is namespaced properly for github org?

Reply:
it's not. Will fix it, thanks.